src/modules/75-cd/components/PipelineSteps/Common/Terraform/Editview/TerraformConfigForm.tsx

```diff
@@ -0,0 +1,444 @@
+                          <Button
+                            icon="question"
+                            minimal
+                            tooltip={`${ConnectorMap[selectedConnector]} ${getString('connector')}`}
+                            iconProps={{ size: 14 }}
+                          />
+                        </Text>
+                      }
+                      type={ConnectorMap[selectedConnector]}
+                      width={400}
+                      name={
+                        isTerraformPlan
+                          ? 'spec.configuration.configFiles.store.spec.connectorRef'
+                          : 'spec.configuration.spec.configFiles.store.spec.connectorRef'
+                      }
+                      placeholder={getString('select')}
+                      accountIdentifier={accountId}
+                      projectIdentifier={projectIdentifier}
+                      orgIdentifier={orgIdentifier}
+                      style={{ marginBottom: 10 }}
+                      multiTypeProps={{ expressions, allowableTypes: filteredTypes }}
+                      multitypeInputValue={multitypeInputValue}
+                    />
+                    <Button
+                      className={css.newConnectorButton}
+                      variation={ButtonVariation.LINK}
+                      size={ButtonSize.SMALL}
+                      disabled={isReadonly}
+                      id="new-config-connector"
+                      text={newConnectorLabel}
+                      icon="plus"
+                      iconProps={{ size: 12 }}
+                      onClick={() => {
+                        /* istanbul ignore next */
+                        setConnectorView(true)
+                        /* istanbul ignore next */
+                        nextStep?.({ formValues: data, selectedType: selectedConnector })
+                      }}
+                    />
+                  </Layout.Horizontal>
+                )}
+              </div>
+              <Layout.Horizontal spacing="xxlarge">
+                <Button
+                  variation={ButtonVariation.PRIMARY}
+                  type="submit"
+                  text={getString('continue')}
+                  rightIcon="chevron-right"
+                  onClick={() => {
+                    /* istanbul ignore next */
+                    nextStep?.({ formValues: formik.values, selectedType: selectedConnector })
+                  }}
+                  disabled={disabled || disableArtifactory}
+                />
+              </Layout.Horizontal>
+            </Form>
+          )
+        }}
+      </Formik>
+    </Layout.Vertical>
+  )
+}
+
+interface TerraformConfigStepTwoProps {
+  allowableTypes: MultiTypeInputType[]
+  isReadonly: boolean
+  onSubmitCallBack: any
+  isTerraformPlan?: boolean
+}
+
+export const TerraformConfigStepTwo: React.FC<StepProps<any> & TerraformConfigStepTwoProps> = ({
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232248)
```suggestion
export const TerraformConfigStepTwo: React.FC<StepProps<unknown> & TerraformConfigStepTwoProps> = ({
```